web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/Gallery/page/ImageForm/ImageDisplay/index.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from "next/image";
+
+export const ImageDisplay = (props: {
+  src: string;
+  type: string;
+  className?: string;
+  width?: number;
+  height?: number;
+}) => {
+  const { src, type, className, width, height } = props;
+  if (type === "verified") {
+    return <img src={src} alt="verified" className={className} />;
```

Comment:
We use img here since verified images are cached by our CDN and we noticed that they ended up loading slower as a result of that working with Next Images